src/components/for-ui-core/FlexContainer.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import propTypes from 'prop-types'
+
+const FlexContainer = ({
```

Comment:
I'm very much against styling by props.
It'll cause a separation of styles to two place:
1. css/style files
2. component code

I'd rather have default styles and accept a `className` so styles can be overriden if needed.
It will also allow to use `display: block` when needed (i. e. sometimes you want a completely different behavior on mobile devices)

Furthermore, using a `className` prop is standard in `ui-core`, it'll reduce complexity of this component, will decrease maintenance amount in case of css standards/browser-implementation changes & allows for better customization when deaing with exception-cases.

I don't really mind this here in `ui-forms`, but when moving this one to `ui-core`, we either use `className` or we think about whether we need this component at all, as it's not an atom or molecule but a mere building block.

Reply:
Managed to get rid of most of these helper components